Tests/UnitTests/Onboarding/MainCoordinatorTests.swift

```diff
@@ -29,10 +28,20 @@ final class MainCoordinatorTests: XCTestCase {
 }
 
 extension MainCoordinatorTests {
-    func test_MainCoordinatorStart() throws {
+    func test_mainCoordinatorStart() throws {
         XCTAssertTrue(navigationController.viewControllers.count == 0)
         sut.start()
         XCTAssertTrue(navigationController.viewControllers.count == 1)
         XCTAssert(navigationController.topViewController is IntroViewController)
     }
+    
+    func test_mainCoordinatorOpensSubCoordinator() throws {
+        sut.start()
```

Comment:
Question for Elishka really:
Should we add any ACs in comments or anything within these tests to make it clear what they are doing?
This is only our first test and I'm not sure clear (from a user's perspective) on what it is testing at first glance.

Reply:
Yes that would be very helpful in general. On the back end, we typically do this in a test name with a convention along the lines of "when I do something, then it returns xyz". We could do that or comments - whatever people prefer in an iOS context.

Reply:
I can update all the tests to include comments in the format you've mentioned @elishkaflint 🙂